### ISSUE TITLE
OpenSSL certificates for LL-HLS invalid

### ISSUE THREAD

**Issue by pietmlr** [aler9/rtsp-simple-server#1005]
v19.2.0
Running OS:
- [X] Linux arm64 standard

## Describe the issue
The javascript console in the browser (safari) says that the certificate for the server publishing LL-HLS is invalid making it a security risk.

## Describe how to replicate the issue
To replicate this issue simply generate the key and the certificate as descriped in README.md 
1. Generate OpenSSL certificate and key just like in the documentation
2. Enable LL-HLS exactly like in the documentation
3. start the server
4. publish any stream to view in LL-HLS (I am using an gstreamer rtsp stream)
5. read with a browser by also follwing the steps in the documentation (i am using the html iframe-tag)

At step 1. you are prompted to enter some information for the certificate, do you know which information is required as I am suspecting the issue being right here. (I also recommend mentioned the inforamtion promt in your documentation with maybe one simple sentence).

**Comment by aler9** [comment]
Hello, this is perfectly normal, as a self-signed certificate isn't trusted by browsers by default. If you need a valid certificate, you have to:

1. buy a domain (i.e. yourdomain.com)
2. make sure that your instance of RTSPSS is accessible to the outside world and from that domain, i.e. http://yourdomain.com:8888/mystream
3. buy a certificate from an authority or generate one by using Let's Encrypt

**Comment by pietmlr** [comment]
but how would i implement the lets encrypt certification and key into rtsp-simple-server? because i tried it providing the path to both in the yaml file but it still wont work

**Comment by aler9** [comment]
Setup is exactly the same as the one of a HTTP server; if you're unable to follow the procedure, I suggest you to try Caddy (https://caddyserver.com), a reverse proxy with automatic Let's Encrypt support. Use Caddy to forward requests to rtsp-simple-server, and use a self-signed certificate inside the configuration of rtsp-simple-server. Browsers will be able to access rtsp-simple-server through Caddy, and they'll see a valid certificate.

**Comment by pietmlr** [comment]
Well, I did all those steps, everything is working (HLS accessible to the outside world, domain working with SSL from Lets Encrypt) I just do not know what to put in the server.key, server.crt rows in the configuration file as i am using a “real” certificate from lets encrypt and not some self generated one. I actually followed the procedure and in the documentation the case of not self assigned certificates is not covered.
Btw, I appreciate your project, impressive work so far!
